locales/te-IN

```diff
@@ -0,0 +1,21 @@
+title = ఫైర్ఫాక్స్  క్వాంటం విడుదల ప్రచారం
```

Comment:
can you please rename the file to te-IN.ftl (see the file format)

Reply:
Yes, doing it now.